docs/DatabaseCatalogAPI.yaml

```diff
@@ -373,7 +373,7 @@ components:
           description: The date and time the dataset was downloaded from the producer, in ISO 8601 format.
           type: string
           example: 2023-07-10T22:06:00Z
-          format: datetime
+          format: date-time
```

Comment:
This is the proper [OpenAPI format type](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#data-types)